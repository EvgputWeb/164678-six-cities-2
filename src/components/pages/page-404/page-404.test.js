import React from 'react';
import renderer from 'react-test-renderer';
import Page404 from './page-404';
import {MemoryRouter} from 'react-router-dom';


it(`Page404 renders correctly`, () => {
  const page404 = renderer
    .create(
        <MemoryRouter>
          <Page404 />
        </MemoryRouter>
    )
    .toJSON();

  expect(page404).toMatchSnapshot();
});
