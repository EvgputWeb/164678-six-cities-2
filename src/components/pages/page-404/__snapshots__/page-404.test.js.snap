// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page404 renders correctly 1`] = `
Array [
  <h1>
    404
  </h1>,
  <p>
    Page not found
  </p>,
  <a
    href="/"
    onClick={[Function]}
  >
    Go to main page &gt;&gt;&gt;
  </a>,
]
`;
